services/kimi_bridge_server/README.md

```
@@ -0,0 +1,103 @@
+export KIMI_BRIDGE_TOKEN=$(python -c "import secrets; print(secrets.token_hex(32))")
+
+# Start the service
+KIMI_BRIDGE_TOKEN=$KIMI_BRIDGE_TOKEN \
+  uvicorn services.kimi_bridge_server.app:app --host 0.0.0.0 --port 8011
+```
+
+## Connecting to the Main Backend
+
+Set these environment variables on the **main backend** service:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)